client/internal/peer/stdnet.go

```diff
@@ -3,9 +3,9 @@
 package peer
 
 import (
-	"github.com/pion/transport/v2/stdnet"
+	"github.com/netbirdio/netbird/client/internal/stdnet"
 )
 
 func (conn *Conn) newStdNet() (*stdnet.Net, error) {
-	return stdnet.NewNet()
+	return stdnet.NewNet(nil, conn.config.InterfaceBlackList)
```

Comment:
In this construction make no sense for two different stdnet.go and stdnet_android.go Both of them doing the same thing at the end. 
I recommend you to create two constructor in the stdnet pkg. On of them with blacklist and one of them with discoverer and with blacklist. In this case in the UpdateInterfaces function you can avoid the if/else statement because you can create two separated function.